promise-test.js

```diff
@@ -3,18 +3,42 @@ require('./lib/promise');
 
 var gun = new Gun();
 
-gun.get('mark').put({
```

Comment:
Oh good! I see that this has already been removed from root of project already :)

Reply:
I did run the test before I deleted. Obviously I changed things slightly (like using gun's wire spec of `get` and `put` instead of `key` and `val`, but yeah. It worked for both of them in the contexts that it made sense for them to work).

👍 👍 👍  job for having tests. We can always stick them in `test/` folder, maybe as `test/extended.js` or something? More tests the better... well, I should clarify: more tests that can be run on the side, the better. I don't want to have to worry about Travis always breaking on trivial things, especially since I require Travis to run older versions of NodeJS that may not support "new" things that some extended API features depend upon. Etc. etc., but that is another discussion.

Reply:
Right, yes, that is a challenge if you need Travis to run some tests on all versions of Node but others to only run on some. Hmmm interesting.